locales/en.js

```diff
@@ -8,6 +8,8 @@ const en = {
   logout: 'Logout',
   changeUser: 'Change User',
   refreshed: 'Refreshed',
+  notProAlertTitle: '無存取權限!',
```

Comment:
This should be in English.

Change key to 'privilegedAccessTitle', value to 'This is a premium feature'